examples/webgl_materials_wireframe.html

```diff
@@ -124,7 +125,7 @@
 						alphaToCoverage: true // only works when WebGLRenderer's "antialias" is set to "true"
 
 					} );
-					material2.extensions.derivatives = true;
+					//material2.extensions.derivatives = true; // is this still needed?
```

Comment:
not sure about the current status of this...

Reply:
I would imagine it's still needed for WebGL 1?